schemas/logs/lacework/events.yml

```diff
@@ -65,13 +65,13 @@ fields:
                         type: string
                       - name: HAS_EXTERNAL_CONNS
                         description: HasExternalConns field
-                        type: int
+                        type: bigint
```

Comment:
It is not allowed in general. Can you provide more context why we need to make this change?

Reply:
The previous `int` was a mistake from before we fixed Golang's `int` to be `int64`.
Leaving it to `int` would mean that the Glue column would not be backwards compatible (int64 -> int32)